Apps/APN/ios/Podfile

```diff
@@ -0,0 +1,43 @@
+require_relative '../node_modules/react-native/scripts/react_native_pods'
+require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+
+platform :ios, '13.0'
+install! 'cocoapods', :deterministic_uuids => false
+
+target 'SampleApp' do
+  config = use_native_modules!
+
+  # Flags change depending on the env values.
+  flags = get_default_flags()
+
+  pod 'CustomerIO/MessagingPushAPN', '2.1.2'
+
+  use_react_native!(
+    :path => config[:reactNativePath],
+    # Hermes is now enabled by default. Disable by setting this flag to false.
+    # Upcoming versions of React Native may rely on get_default_flags(), but
+    # we make it explicit here to aid in the React Native upgrade process.
+    :hermes_enabled => true,
+    :fabric_enabled => flags[:fabric_enabled],
+    # Enables Flipper.
+    #
+    # Note that if you have use_frameworks! enabled, Flipper will not work and
+    # you should disable the next line.
+    # An absolute path to your application root.
+    :app_path => "#{Pod::Config.instance.installation_root}/.."
+  )
+end
+
+target 'NotificationServiceExtension' do
+  pod 'CustomerIO/MessagingPushAPN', '2.1.2'
```

Comment:
```suggestion
  pod 'CustomerIO/MessagingPushAPN', '~> 2'
```

Reply:
I see this now lol. Have already made fix in [this commit.](https://github.com/customerio/customerio-reactnative/pull/136/commits/008d7bfb040cace131b77a9c8516bd64fbdc5831)